Formula/atlassian-cli.rb

```diff
@@ -1,9 +1,9 @@
 class AtlassianCli < Formula
   desc "Command-line interface clients for Atlassian products"
   homepage "https://bobswift.atlassian.net/wiki/pages/viewpage.action?pageId=1966101"
-  url "https://bobswift.atlassian.net/wiki/download/attachments/16285777/atlassian-cli-5.7.0-distribution.zip"
-  version "5.7.0"
-  sha256 "7d1af9dd7b5fe0fa35ba13f24d5d98fa49e0146bbae2181c43d845f3bf93ad2f"
+  url "https://bobswift.atlassian.net/wiki/download/attachments/16285777/atlassian-cli-6.2.0-distribution.zip"
+  version "6.2.0"
```

Comment:
- Java 1.8 is required for CLI 6.0 and above.
- The commit message should be a single line saying `atlassian-cli 6.2.0`, with no further comments (which can go in the text of the pull request).